error_codes.txt

```diff
@@ -0,0 +1,8 @@
+type errorResponse struct {
+	Error_code    int    `json:"error_code"`
+	Error_message string `json:"error_message"`
+}
+
+100, "All the fields - name, price and inventory - must be filled correctly."
+
+101, "There is already a book with this name on database:"
```

Comment:
Whats the reasoning behind starting in 100?

Reply:
Just following the pattern in http status code, where the first algarism could indicate a parent class.